tests/urls.py

```diff
@@ -9,4 +14,6 @@
     url(r'^api/', include('osmaxx.excerptexport.rest_api.urls', namespace='excerptexport_api')),
     url(r'^job_progress/', include('osmaxx.job_progress.urls', namespace='job_progress')),
     url(r'^pages/', include('osmaxx.core.urls', namespace='pages')),
+    # we're serving through nginx in production, but we need this here so we can test more easily!
+    url(r'^%s(?P<path>.*)$' % re.escape(settings.MEDIA_URL.lstrip('/')), serve, kwargs={'document_root': settings.MEDIA_ROOT}),
```

Comment:
I'm merely using the same syntax as in the django documentation, I think that is OK and more easy to find that way, when searching for [it](https://docs.djangoproject.com/en/1.9/ref/contrib/staticfiles/#static-file-development-view) on the web